### DISCUSSION TITLE
why do we need to press <CR> when moving the focus?

### DISCUSSION THREAD

**Discussion by azinsharaf** [zellij-org/zellij#2442] [Q&A]
when moving the cursor focus (for example from left panel to the ringht one) we use `CTRL+ P + L + <cr> `. Is there any reason we need to press enter? I see it as an axtra step and wondering that part can be removed?

**Comment by Lockszmith-GH** [comment]
It's a guess, but if you have more than one pane, the idea is that since you had to use a prefix key (`Ctrl`+`P`), you would like to perform all the movements and then break.

If you don't like that - I'm pretty sure you can configure the binding in the `config.kdl` file.

**Reply by azinsharaf** [reply]
@Lockszmith-GH  yeah makes sense. Do you know how i can configure that binding?

**Reply by imsnif** [reply]
Check out https://zellij.dev/documentation/configuration.html (and more specifically the keybindings section) for a detailed explanation.

**Reply by Lockszmith-GH** [reply]
I use the Alt+arrows to move across panes, have you tried that?
